src/providers/toast.service.ts

```diff
@@ -14,4 +18,24 @@ export class ToastService {
     this.faveToast = this.toast.create({message: txt, position: 'bottom', showCloseButton: true});
     this.faveToast.present();
   }
+  locationToast():void{
+    let txt = 'Unable to retrieve current location';
+    this.noLocationToast = this.toast.create({message: txt, position: 'bottom', showCloseButton: true});
+    this.noLocationToast.present();
+  }
+  directionToast():void{
+    let txt = 'Cannot get directions to this stop. Please ensure location services are enabled.';
+    this.directToast = this.toast.create({message: txt, position: 'bottom', showCloseButton: true});
+    this.directToast.present();
+  }
+  statusDirection(status: string): void{
+    let txt = `Couldn't get directions to this stop. Status code ${status}`;
```

Comment:
You should probably use single quotes instead of back ticks. 👍

Reply:
Or double quotes since you have a single quote in there.

Reply:
Backticks allow for string interpolation (the `${status}`) without needing to do `'some string' + status + 'anotherString'` - I'm in favor!

Reply:
Ah, okay. I hate it, but okay.